api/test/wfl/unit/modules/all_test.clj

```diff
@@ -11,6 +11,6 @@
   (let [edn          (wfl/get-the-version)
         has-version? #(= (:release %) (-> % :path basename edn))]
     (is (every? has-version? [aou/workflow-wdl
-                              #_sg/workflow-wdl ; Now a commit hash. =tbl
+                              sg/workflow-wdl
```

Comment:
I think it's still a commit hash? https://github.com/broadinstitute/wfl/blob/adac9d5ab0a6872dbabd60697b6400b65b476b67/api/src/wfl/module/sg.clj#L23 @tbl3rd when/ shall we do this here?

Reply:
Yeah. We do this for wgs as well. I'm not sure why we even include this in "the version".

Reply:
I think for WGS, the reason is cauz I double-submitted what's hot-fixed (which was a commit than a released wdl) in production. I can't remember why we use commit hash for sg, might be that it's still under testing?

Reply:
Yep. A working WDL hasn't been released yet.